server/sonar-web/src/main/js/api/permissions.ts

```diff
@@ -178,12 +178,16 @@ export function removeProjectCreatorFromTemplate(
   return post('/api/permissions/remove_project_creator_from_template', { templateId, permission });
 }
 
+export interface PermissionUser {
+  login: string;
+}
+
 export function getPermissionsUsersForComponent(
   projectKey: string,
   query?: string,
   permission?: string,
   organization?: string
-): Promise<any> {
+): Promise<PermissionUser[]> {
```

Comment:
Currently `PermissionUser` isn't used anywhere else. Why not just `Promise<{ login:string }[]>` ?